collect/central_collector.go

```diff
@@ -410,7 +408,7 @@ func (c *CentralCollector) processTraces(ctx context.Context) error {
 		return nil
 	}
 
-	statuses, err := c.Store.GetStatusForTraces(ctx, ids)
+	statuses, err := c.Store.GetStatusForTraces(ctx, ids, centralstore.DecisionKeep, centralstore.DecisionDrop)
```

Comment:
Should the default case below now error? This function should no longer ever return something that isn't Keep or Drop